src/tasks/FountainTask.py

```
@@ -0,0 +1,251 @@
+
+    def do_run(self):
+        last_error = None
+        for attempt in range(1, self.TASK_RETRY_COUNT + 2):
+            self._fountain_task_start = time.time()
+            self.log_info(f"attempt {attempt}/{self.TASK_RETRY_COUNT + 1}")
+            try:
+                if self.run_fountain_flow():
+                    return True
+            except TaskDisabledException:
+                raise
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)